platform-sdk/docs/proposals/TSS-Ledger-Id/tss-ledger-id.md

```diff
@@ -0,0 +1,73 @@
+## Interactions
+
+| Interaction       | Entities                     | 
+|-------------------|------------------------------|
+| Designers         | Edward, Kore, Anthony, Artem | 
+| Implementers      | team-spacetime               |
+| Team Impacts      | Services, DevOps             |
+| Related Proposals | TSS-Library, TSS-Block-Proof |
+| HIPS              | HIP-???                      |
+
+# Context
+
+To support the ability to so block proofs, we need the ability to assign a public/private key-pair to the network or ledger itself.  The following requirements on the ledger key must hold: 
+* The public key of the ledger must not change when the composition of network nodes change. 
+* The private key of the ledger must not be known by anyone. 
+* The signature of the ledger private key signing a message must be producible from a threshold of network nodes signing the message. 
+* Verification of the ledger signature must be cheap for an EVM smart contract. 
+
+## Design Decisions
+
+Describe the high level options available, the trade-offs between them and the decisions made to commit to one option
+and its opportunity costs.
+
+# Changes
+
+## Public API
+
+Describe any public API changes or additions. Include stakeholders of the API. Mark this section as N/A if not
+applicable.
+
+Code can be included in the proposal directory, but not submitted to the code base.
+
+## Core Behaviors
+
+Describe any new or modified behavior. Mark this section as N/A if not applicable.
+
+### Metrics
+
+How is the behavior observable in metrics?
+
+## Performance
+
+Describe any expected performance changes. This section is mandatory for platform wiring changes. Mark this section as
+N/A if not applicable.
```

Comment:
I dislike to have multiple sections with N/A in it. Better remove the section in that case.